Dockerfile.staging

```diff
@@ -4,5 +4,5 @@ COPY package*.json ./
 RUN apk add --no-cache git python3 make g++ ffmpeg
 RUN npm install && npm install typescript -g
 COPY . .
-RUN tsc
+RUN tsc -p tsconfig.json
```

Comment:
Isn't this the default path?

Reply:
This using default typescript settings unless a file is specified would explain why the build failed on staging but passed the CI typecheck

Reply:
[The CI](https://github.com/spknetwork/acela-core/blob/main/.github/workflows/node.js.yml#L29-L32) looks a lot different than the Dockerfile.

Also, maybe it's the `npm install typescript -g` this would install the latest version of typescript while the CI does not.

Reply:
if you look at the package json, `npm run tsc:noemit` is actually `npm run tsc -p tsconfig.json --noEmit` under the hood.

Reply:
My original point was isn't running `tsc` equivalent to `tsc -p tsconfig.json` since "tsconfig.json" is the default path?

Reply:
The non-prod docker file uses `RUN npm run tsc:noemit` and ran without any type errors

Reply:
I think that part is a red-haring, since `tsc` and `tsc -p tsconfig.json` should be equivalent. Please consider my previous point:
> Also, maybe it's the npm install typescript -g this would install the latest version of typescript while the CI does not.

So, the different versions of typescript can cause different type checking behaviors.

With these 2 points, I think this change does not fix the issue, but removing `npm install typescript -g` from the Dockerfile should.

Reply:
Looks like you have included the requested change in #30, so I think we can mark this as resolved.